manifests/profile/copymod.pp

```diff
@@ -1,4 +1,4 @@
-define bootstrap::copymod(
```

Comment:
Oops, I missed that one.  I don't think we're actually using this so I may just pull it altogether.